### DISCUSSION TITLE
Ability to copy text from note view screen

### DISCUSSION THREAD

**Discussion by The-Ladle** [ostoaleonardo/notes#1] [Feature requests]
Normally, you have to press edit to copy note text, but this will allow notes to be copied from the note view. Someone already suggested this in Discord but I am putting it here so it is more organized.

**Comment by ostoaleonardo** [comment]
Do you think the view screen is necessary? I will add the autosave feature.

**Reply by The-Ladle** [reply]
> Do you think the view screen is necessary? I will add the autosave feature.

With autosave, probably not
